prime-angular-frontend/src/app/modules/health-auth-site-reg/health-auth-site-reg-routing.module.ts

```diff
@@ -112,13 +103,7 @@ const routes: Routes = [
             path: HealthAuthSiteRegRoutes.HEALTH_AUTH_CARE_TYPE,
             component: HealthAuthCareTypePageComponent,
             canDeactivate: [CanDeactivateFormGuard],
-            data: { title: 'Health Authority Care Type' }
-          },
-          {
-            path: HealthAuthSiteRegRoutes.SITE_ADDRESS,
-            component: SiteAddressPageComponent,
-            canDeactivate: [CanDeactivateFormGuard],
-            data: { title: 'Site Address' }
+            data: { title: 'Health Authority Care Type & Vendor' }
```

Comment:
Mock-up splits into two sub-titles

![image](https://user-images.githubusercontent.com/7586665/211422903-1ce965bb-6316-426e-bdd2-90b1d5788ac3.png)

Reply:
this is fixed.